### PR TITLE
Fix 'Embedding Null Code' title and amend body text

### PR DESCRIPTION
**Pull request by ADubhlaoich**
I was catching up on OWASP news and noticed the new website - this fixes a typo in a page title, though while I was making the change I also re-phrased a lot of the wording on the page, as it was unclear or redundant.

I also attempted to fix some of the spacing issues with the preformatted code examples, as they appeared to be missing proper linebreaks. I'm willing to do a lot of this for other pages and also clear out all of the existing issues in the repository.

### CODE REVIEW

File: pages/attacks/Embedding_Null_Code.md [hunk 1]
```diff
@@ -14,65 +14,53 @@ permalink: /attacks/Embedding_Null_Code
 
 The Embedding NULL Bytes/characters technique exploits applications that
 don’t properly handle postfix NULL terminators. It is used as a
-technique to perform other attacks, like directory browsing, path
-traversal, SQL injection, execution of arbitrary code, among others. It
-can be found lots of vulnerable applications and exploits available to
-abuse systems using this technique.
+technique to perform other attack such as directory browsing, path
```
Comment (kingthorin) [inline pages/attacks/Embedding_Null_Code.md]: attacks should still be plural

File: pages/attacks/Embedding_Null_Code.md [hunk 2]
```diff
@@ -14,65 +14,53 @@ permalink: /attacks/Embedding_Null_Code
...
+ (pdf.ocx) to allow remote code execution.
 
-In this case, the request must be made to a web server that truncates
-the request at the null byte (%00), as Microsoft IIS and Netscape
-Enterprise web servers. Though the requested URI is truncated for the
-purposes of locating the file, the long string is still passed to the
-Adobe ActiveX component. This component triggers a buffers overflow
-within RTLHeapFree() allowing for an attacker to overwrite an arbitrary
-word in memory.
+The attack occurs  when a link is requested as follows:
```
Comment (kingthorin) [inline pages/attacks/Embedding_Null_Code.md]: Could you fix the dbl space before ‘when’?

File: pages/attacks/Embedding_Null_Code.md [hunk 3]
```diff
@@ -14,65 +14,53 @@ permalink: /attacks/Embedding_Null_Code
...
 
-This attack can be performed by adding malicious content to the end of
-any embedded link and referencing any Microsoft IIS or Netscape
-Enterprise web server. It is not necessary to establish a malicious web
-site to execute this attack.
+`GET /some_dir/file.pdf.pdf%00[long string] HTTP/1.1`  
 
-For more details about this attack, see External References topics.
+This exploit can be used on against a web server that truncates the request at
```
Comment (kingthorin) [inline pages/attacks/Embedding_Null_Code.md]: ‘on’ or ‘against’ not both

File: pages/attacks/Embedding_Null_Code.md [hunk 4]
```diff
@@ -14,65 +14,53 @@ permalink: /attacks/Embedding_Null_Code
...
+This exploit can be used on against a web server that truncates the request at 
+the null byte (%00), such as Microsoft IIS and Netscape Enterprise web servers. 
+Though the requested URI is truncated when locating the file, the full string 
+is still passed to the Adobe ActiveX component. It then triggers a buffer 
+overflow within RTLHeapFree(), allowing an attacker to overwrite arbitrary
+memory.
 
+It can be performed by adding malicious content to the end of any embedded link 
+that references a Microsoft IIS or Netscape Enterprise web server. 
 ## External References
```
Comment (kingthorin) [inline pages/attacks/Embedding_Null_Code.md]: Could you add a blank line before the heading?

File: pages/attacks/Embedding_Null_Code.md (hunk 1 above)
Comment (ADubhlaoich) [reply pages/attacks/Embedding_Null_Code.md]: Fixed, though I made another adjustment to the start of the sentence.

File: pages/attacks/Embedding_Null_Code.md (hunk 2 above)
Comment (ADubhlaoich) [reply pages/attacks/Embedding_Null_Code.md]: Fixed

File: pages/attacks/Embedding_Null_Code.md (hunk 3 above)
Comment (ADubhlaoich) [reply pages/attacks/Embedding_Null_Code.md]: I don't actually know if I screwed up or if that was in the original text, but fixed.

File: pages/attacks/Embedding_Null_Code.md (hunk 4 above)
Comment (ADubhlaoich) [reply pages/attacks/Embedding_Null_Code.md]: Done.

**Comment by ADubhlaoich** [comment]
I'll push the amendments momentarily; I'm going to take a look at where the page was originally migrated from to try source the original author & any contributors.

**Comment by kingthorin** [comment]
Thanks, is this ready for review/merge again?

**Comment by ADubhlaoich** [comment]
Yup!

**Pull request merged**